### DISCUSSION TITLE
How can I sync to subfolder?

### DISCUSSION THREAD

**Discussion by gerroon** [remotely-save/remotely-save#386] [Q&A]
Hi

I am using Nextcloud as my webdav backend. My notes are under "text/notebooks" so they are not at the root. I there a way to sync to that folder. It would be quite a sucky situation if I have to move my notes to the root like "notebooks" because my notebooks are accessed by bunch of users and apps. 

Is this possible?

**Comment by fyears** [comment]
could you try to directly set your dav address to the full path? like http://example.com/dav/text/textbooks

**Reply by shanmukhuggini** [reply]
anyone working on google drive based one just like one drive ? I'm keen to give an hand

**Comment by gerroon** [comment]
Hi

That does not work with Nextcloud unfortunately, setting a subfolder like that gives user password permission error.

The problem is that this plugin seems to have an issue with folder separator

![image](https://github.com/remotely-save/remotely-save/assets/8519469/f981f0e6-4384-4e49-ae83-ecb548e9e84f)

**Comment by fyears** [comment]
set in the **webdav address**, not root folder

**Comment by gerroon** [comment]
"That does not work with Nextcloud unfortunately, setting a subfolder like that gives user password permission error."

**Comment by fyears** [comment]
hmmmm.... fine, so i will take it as a feature request.

let me implement the feature in the near future

**Comment by gerroon** [comment]
Thanks for considering it.

**Comment by fyears** [comment]
the latest version makes it possible

**Comment by HastagGuigui** [comment]
I'm on latest and this doesn't seem to work anymore... I'm just getting hit with a 403 forbidden even though not adding the subfolder works fine (but puts the files in a folder in the root of the nextcloud user filesystem)

**Comment by JacobDB** [comment]
I get a "405: Method Not Allowed" message when attempting to set the server path as described for Nextcloud, would really appreciate being able to put my vaults in a subdirectory.
